conanfile.py

```diff
@@ -196,13 +198,17 @@ def package_info(self):
             self.cpp_info.includedirs.append(os.path.join(self.build_folder, "include"))
             bindir = os.path.join(self.build_folder, "bin")
             luadir = os.path.join(self.source_folder, "engine/lua")
-            pydir = os.path.join(self.source_folder, "python/python_api")
+            pydir = os.path.join(self.build_folder, "lib/cloe/python")
+            py_path = os.path.join(self.source_folder, "python/python_api")
```

Comment:
`pydir` and `py_path` are basically the same word. Can we distinguish these two paths better? `pypath_source` and `pypath_build` or something like that.